src/main/kotlin/net/lumalyte/lg/infrastructure/persistence/invitations/GuildInvitationRepositorySQLite.kt

```
@@ -0,0 +1,196 @@
+    override fun removeOlderThan(olderThan: Long): Int {
+        return try {
+            val cutoffTime = Instant.ofEpochSecond(olderThan)
+
+            database.executeUpdate(
+                """
+                DELETE FROM guild_invitations
+                WHERE invited_at < ?
+                """.trimIndent(),
+                cutoffTime.toString()
+            )
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)